packages/d3fc-webgl/test/shaders/line/shaderSpec.js

```diff
@@ -0,0 +1,39 @@
+import * as fc from '../../../index';
+import { expectVertexShader, getShaders } from '../../util/vertextShaderUtil';
+
```

Comment:
Could you add more test cases?

- 1st point
- 2nd point
- A middle point
- Last point